internal/mtail/examples_integration_unix_test.go

```diff
@@ -0,0 +1,172 @@
+// Copyright 2011 Google Inc. All Rights Reserved.
+// This file is available under the Apache license.
+
+//go:build unix
+// +build unix
+
+package mtail_test
+
+// Two mtails both alike in dignity.
```

Comment:
Pop a note here like you did for the other tests saying that Pipes are unix only thing.

Reply:
done